app/_layout.tsx

```diff
@@ -1,27 +1,30 @@
 import '../global.css';
 
 import {
-  useFonts,
   DMSans_400Regular,
   DMSans_400Regular_Italic,
   DMSans_500Medium,
   DMSans_500Medium_Italic,
   DMSans_700Bold,
   DMSans_700Bold_Italic,
 } from '@expo-google-fonts/dm-sans';
+
 import {
   DMSerifText_400Regular,
   DMSerifText_400Regular_Italic,
 } from '@expo-google-fonts/dm-serif-text';
-import { PrivyProvider } from '@privy-io/expo';
+
+import { Inter_400Regular, Inter_500Medium, Inter_600SemiBold } from '@expo-google-fonts/inter';
+
+import { PrivyProvider, PrivyElements } from '@privy-io/expo';
 import { Drawer } from 'expo-router/drawer';
 import { GestureHandlerRootView } from 'react-native-gesture-handler';
+import { useFonts } from 'expo-font';
```

Comment:
Is there a reason why you are using `useFonts` from `expo-font` and not `expo-google-fonts/dm-sans` ?

Reply:
I think that the `useFont` method is the same from all the packages
But it's cleaner that way, just wanted to know